src/edge/tests/test_annotations.py

```diff
@@ -359,3 +359,35 @@ def test_inherits_new_annotations_on_inserted_fragment(self):
         self.assertEquals(f.annotations()[0].feature.name, 'X1')
         self.assertEquals(f.annotations()[0].feature_base_first, 1)
         self.assertEquals(f.annotations()[0].feature_base_last, 3)
+
+    def test_annotate_multiple_chunks_with_same_annotation(self):
+        self.root.annotate_chunks(((1, 3), (7, 9)), 'A1', 'gene', 1)
```

Comment:
how would we handle isoforms of a gene? Do we have support for 2 level annotations or extra information when the annotation is like `NM.00001` and the gene is `foo`?

Reply:
you can provide a qualifiers argument to annotation chunks method and annotate method. the view test shows this.